README.md

```diff
@@ -81,7 +81,38 @@ The following tools are required for development:
   Sequencescape (currently 5.7) use [this](https://gist.github.com/operatino/392614486ce4421063b9dece4dfe6c21)
   helpful link.
 
-## Getting started
+## Getting started (using Docker)
+
+To set up a local development environment in Docker, you have to build a new Docker image for
+Sequencescapek. start a stack of services that include a mysql database, and reset
+this database contents. You can do all together by running the command:
+
+```
+RESET_DATABASE="true" docker-compose up
```

Comment:
I don't think the quotes are necessary in this environment variable.  The command line will interpret everything up to the first whitespace as a string to assign to the variable.

```suggestion
RESET_DATABASE=true docker-compose up
```